src/Events/Repository/CloudEventRepository.cs

```diff
@@ -49,6 +51,33 @@ public async Task CreateAppEvent(CloudEvent cloudEvent)
             pgcom.Parameters.Add(new NpgsqlParameter("cloudevent", cloudEvent.Serialize()) { Direction = System.Data.ParameterDirection.Input });
 
             await pgcom.ExecuteNonQueryAsync();
+
+            await using NpgsqlCommand pgcom2 = new NpgsqlCommand(insertEventSql, conn)
+            {
+                Parameters =
```

Comment:
Didn't know this syntax was an option. The Parameters property is read only. What is the result of this syntax? Is the new parameter added to existing collection or does it create a new collection?

Reply:
Fun stuff:
https://sharplab.io/#v2:EYLgtghglgdgNAExAagD4AEBMBGAsAKHQAYACdbAFgG4CD0BmMzEgYRIG8CTuzH0KSAWQAUASg5ceUlgHsANnICmAYwAuUGTAAqigM6rFAJxIH9RkgF4SMRQHcxnfFOc8AMlH2WSkl1Me+AkgAiU1Ugn0CAXwipSJonHmiE7gjYA0MYCDkmVnklNQ1tPXSJZOcGEnd9AB5yIgA+EgB9OQ9VeICKqtVa7AbKto4SAHNFdpIkzsZZBRV1TR0zQzFSwJ4WwasbWwGauvqxDt9JxIJIoA===

Reply:
In other words. Elements are added